src/release-creator/main.ts

```diff
@@ -0,0 +1,88 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+
+import { createLogger, createOctokitInstance, getPrefixedThrow } from "../utils";
+
+const repos = [{ repo: "github-workflows", mainBranchName: "develop" }];
+// const repos = [{repo: 'frontend-mono', mainBranch: 'develop'}]
+const getOptionalInput = (name: string) => core.getInput(name) || undefined;
+
+const WORKFLOW_NAME = "release-branch-tagger.yml";
```

Comment:
I expect that the repo's name the reusable workflow file with this exact name, so I can trigger it from this repo across all repos, to trigger a stable tag (eg. `web-admin@2021-Q3`) If this file cant be found, it will be logged, and you'll still be able to trigger it in the specific repo itself by using the workflow_dispatch trigger manually.